src/app/zap-templates/zcl/data-model/silabs/general.xml

```diff
@@ -191,14 +191,14 @@ limitations under the License.
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
     <globalAttribute side="either" code="0xFFFD" value="5"/>
-    <attribute side="server" code="0x0000" define="CURRENT_LEVEL" type="INT8U" min="0x00" max="0xFF" writable="false" reportable="true" default="0x00" optional="false">current level</attribute>
+    <attribute side="server" code="0x0000" define="CURRENT_LEVEL" type="INT8U" isNullable="true" min="0x00" max="0xFF" writable="false" reportable="true" default="0x00" optional="false">current level</attribute>
```

Comment:
If isNullable="true", should max not become "0xFE"?

Reply:
> > I see this was added in the spec as default value. But how can a current level be null? In the spec i only see that is means undefined value.
> > What do you do with your light if you turn it on and the current level is null?
> 
> I believe the intention is that a device comes with null out of the factory, and uses StartUpCurrentLevel on the first start, after that CurrentLevel has been set. @Rob-Houtepen-Signify to confirm.

From the Matter spec:
7.17.4. Default Column
A default value defined in the Default column is not meant to be the value used when the server
returns to factory fresh settings. Specified conformance for data fields may be optional or change
over time. A default value is defined to complete dependencies when the actual data field value is
not present.

So that means that after a light starts up, CurrentLevel should simply reflect the output of the light. This may indeed be determined by StartUpCurrentLevel or when StartUpCurrentLevel == null determined by the preference of the manufacturer.

I agree with Markus that it is a bit difficult to imagine when null is used in practice. But its defined like this and please keep in mind that this cluster can be used for different device types as well.

Reply:
> If isNullable="true", should max not become "0xFE"?

According to the spec :yes. The spec also says default is null, that is also not `default="0x00"`, or? What should that be? 0xFF?

Someone with more experience in SiLabs XML would need to answer that. @jmartinez-silabs?

Reply:
With Rob explanation I would go with default null like you did here. i would also change to reflect the spec max 0xFE